src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/CreateTemporaryTargetAssemblyProject.cs

```diff
@@ -0,0 +1,391 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Collections;
+
+using System.Globalization;
+using System.Diagnostics;
+using System.Reflection;
+using System.Resources;
+using System.Xml;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using MS.Utility;
+using MS.Internal.Tasks;
+
+// Since we disable PreSharp warnings in this file, PreSharp warning is unknown to C# compiler.
+// We first need to disable warnings about unknown message numbers and unknown pragmas.
+#pragma warning disable 1634, 1691
+
+namespace Microsoft.Build.Tasks.Windows
+{
+    #region CreateTemporaryTargetAssemblyProject Task class
+
+    /// <summary>
+    ///   This task is used to generate a temporary target assembly project. 
+    /// 
+    ///   The generated project file is based on current project file, with below
+    ///   modification:
+    /// 
+    ///       A:  Add the generated code files (.g.cs) to Compile Item list.
+    ///       B:  Replace Reference Item list with ReferencePath item list.
+    ///           So that it doesn't need to rerun time-consuming task 
+    ///           ResolveAssemblyReference (RAR) again.
+    /// 
+    /// </summary>
+    public sealed class CreateTemporaryTargetAssemblyProject : Task
+    {
+        //------------------------------------------------------
+        //
+        //  Constructors
+        //
+        //------------------------------------------------------
+
+        #region Constructors
+
+        /// <summary>
+        /// Constructor 
+        /// </summary>
+        public CreateTemporaryTargetAssemblyProject()
+            : base(SR.SharedResourceManager)
+        {
+        }   
+        
+        #endregion Constructors
+
+        //------------------------------------------------------
+        //
+        //  Public Methods
+        //
+        //------------------------------------------------------
+
+        #region Public Methods
+
+        /// <summary>
+        /// ITask Execute method
+        /// </summary>
+        /// <returns></returns>
+        /// <remarks>Catching all exceptions in this method is appropriate - it will allow the build process to resume if possible after logging errors</remarks>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
+        public override bool Execute()
+        {
+            System.Diagnostics.Debugger.Launch();
```

Comment:
You've written a debugging code here. Maybe you should remove it in the product code.